src/login/refresh-token.jsx

```diff
@@ -0,0 +1,11 @@
+const refreshToken = async(handle, refreshTiming) => {
+  const newAuthRes = await handle.reloadAuthResponse()
```

Comment:
Do we really need to await this function? Can't we do our logic within a `.then` callback? (or even better, RXJS pipe!)